vagrant/hosts

```diff
@@ -4,7 +4,7 @@
 
 [all:vars]
 ansible_ssh_user=vagrant
-ansible_dns=mesos.dev
+ansible_domainname=mesos.dev
```

Comment:
And you are right, ansible 2.1 puts wrong data into files, when using ansible_dns, I had to change the name.